frontend/src/components/Election/Voting/RankedBallotView.js

```diff
@@ -24,7 +24,7 @@ export default function RankedBallotView({
       <Typography align='left' component="li">
         Rank the candidates in order of preference.
       </Typography>
-      { race.voting_method === 'RankedRobin' &&
+      { race.voting_method === 'Ranked-Robin' &&
```

Comment:
Ah I see

The ranked robin ballots weren't rendering since there a mix of Ranked-Robin and RankedRobin. Changing it to Ranked-Robin fixed it, but I'll go through and make sure everything's RankedRobin instead

Reply:
I just changed it back and the ballot is still broken on this election http://localhost:3000/Election/ddd19c5c-594d-4f2f-9d4a-a53aa9df8301/vote

So old elections created with Ranked-Robin won't work anymore but new ones should work, and that's probably fine